File: LEETCODE/LEETCODE-CPP/1323.cpp
//1323. Maximum 69 Number
//https://leetcode.com/problems/maximum-69-number/

class Solution {
public:
    int maximum69Number (int num) {
        string s=to_string(num);
        
        int l=s.length(), i=0;
        while(i<l){
            if(s[i]=='6'){
                s[i]='9';
                break;
            }
            i++;
        }
        
        num=stoi(s);
        return num;
    }
};

File: LEETCODE/LEETCODE-JAVASCRIPT/1323.js
// 1323. Maximum 69 Number
// https://leetcode.com/problems/maximum-69-number/description

/**
 * @param {number} num
 * @return {number}
 */

//Method-1 brute force
var maximum69Number  = function(num) {
    let s= num.toString().split('');

    for(let i=0;i<s.length;i++){
        if(s[i]==='6'){
            s[i]='9';
            break;
        }
    }

    return parseInt(s.join(''));
};

//Method-2 one-liner code
var maximum69Number  = function(num) {
    return parseInt(num.toString().replace('6', '9'));
};